Liroy/jenkins_docker_setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+echo "Setting up jenkins Dockerfile..."
```

Comment:
Not a criticism just nice enhancement, you may use end-of-file for insert a multi-line strings :)
 
```
cat << EOF
Multi
Line
Strings
EOF
```